src/o365/spo/commands/list/list-view-list.spec.ts

```diff
@@ -0,0 +1,2260 @@
+                "DefaultView":false,
+                "DefaultViewForContentType":false,
+                "EditorModified":false,
+                "Formats":null,
+                "Hidden":true,
+                "HtmlSchemaXml":"<View Name=\"{3D760127-982C-405E-9C93-E1F76E1A1110}\" Type=\"HTML\" Hidden=\"TRUE\" DisplayName=\"assetLibTemp\" Url=\"/sites/ninja/Shared Documents/Forms/Thumbnails.aspx\" Level=\"1\" BaseViewID=\"40\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/dlicon.png?rev=45\"><ViewFields><FieldRef Name=\"LinkFilename\" /></ViewFields><RowLimit>20</RowLimit><Query><OrderBy><FieldRef Name=\"LinkFilename\" /></OrderBy></Query></View>",
+                "Id":"3d760127-982c-405e-9c93-e1f76e1a1110",
+                "ImageUrl":"/_layouts/15/images/dlicon.png?rev=45",
+                "IncludeRootFolder":false,
+                "ViewJoins":null,
+                "JSLink":null,
+                "ListViewXml":"<View Name=\"{3D760127-982C-405E-9C93-E1F76E1A1110}\" Type=\"HTML\" Hidden=\"TRUE\" DisplayName=\"assetLibTemp\" Url=\"/sites/ninja/Shared Documents/Forms/Thumbnails.aspx\" Level=\"1\" BaseViewID=\"40\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/dlicon.png?rev=45\" ><Query><OrderBy><FieldRef Name=\"LinkFilename\" /></OrderBy></Query><ViewFields><FieldRef Name=\"LinkFilename\" /></ViewFields><RowLimit>20</RowLimit><Toolbar Type=\"None\"/></View>",
+                "Method":null,
+                "MobileDefaultView":false,
+                "MobileView":false,
+                "ModerationType":null,
+                "NewDocumentTemplates":null,
+                "OrderedView":false,
+                "Paged":false,
+                "PersonalView":false,
+                "ViewProjectedFields":null,
+                "ViewQuery":"<OrderBy><FieldRef Name=\"LinkFilename\" /></OrderBy>",
+                "ReadOnlyView":false,
+                "RequiresClientIntegration":false,
+                "RowLimit":20,
+                "Scope":0,
+                "ServerRelativePath":{
+                  "__metadata":{
+                    "type":"SP.ResourcePath"
+                  },
+                  "DecodedUrl":"/sites/ninja/Shared Documents/Forms/Thumbnails.aspx"
+                },
+                "ServerRelativeUrl":"/sites/ninja/Shared Documents/Forms/Thumbnails.aspx",
+                "StyleId":null,
+                "TabularView":true,
+                "Threaded":false,
+                "Title":"assetLibTemp",
+                "Toolbar":null,
+                "ToolbarTemplateName":null,
+                "ViewType":"HTML",
+                "ViewData":null,
+                "VisualizationInfo":null
+              },
+              {
+                "__metadata":{
+                  "id":"https://contoso.sharepoint.com/sites/ninja/_api/Web/Lists(guid'1f187321-f086-4d3d-8523-517e94cc9df9')/Views(guid'1204c10a-978d-46c8-a6f2-65ecfa5e03d5')",
+                  "uri":"https://contoso.sharepoint.com/sites/ninja/_
```

Comment:
You are sending https request with headers `'accept': 'application/json;odata=nometadata'` meaning that you should not have any "__metadata":{ in your mocks as well. 

The easiest way to get the right data is to execute the command with debug option like: 
`o365 spo list view list --webUrl https://velingeorgiev.sharepoint.com --listTitle l2 --debug`
Then find your response data in the  debug output and paste it here. It should look like that 

```
...
Executing web request...
headers: {"authorization":"Bearer eyJ0eXAiOi...","accept":"application/json;odata=nometadata","User-Agent":"NONISV|SharePointPnP|Office365CLI/1.12.0"}
json   : true
method : GET
url    : https://velingeorgiev.sharepoint.com/_api/web/lists/getByTitle('l2')/views

value: [{"Aggregations":null,"AggregationsStatus":null,"BaseViewId":"1","ColumnWidth":null,"ContentTypeId":{"StringValue":"0x"},"CustomFormatter":null,"DefaultView":true,"DefaultViewForContentType":false,"EditorModified":false,"Formats":null,"Hidden":false,"HtmlSchemaXml":"<View Name=\"{BA84217C-8561-4234-AA95-265081E74BE9}\" DefaultView=\"TRUE\" MobileView=\"TRUE\" MobileDefaultView=\"TRUE\" Type=\"HTML\" DisplayName=\"All Items\" Url=\"/Lists/l2/AllItems.aspx\" Level=\"1\" BaseViewID=\"1\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/generic.png?rev=45\"><XslLink Default=\"TRUE\">main.xsl</XslLink><JSLink>clienttemplates.js</JSLink><RowLimit Paged=\"TRUE\">30</RowLimit><Toolbar Type=\"Standard\" /><ViewFields><FieldRef Name=\"LinkTitle\" /></ViewFields><Query><OrderBy><FieldRef Name=\"ID\" /></OrderBy></Query><ParameterBindings><ParameterBinding Name=\"NoAnnouncements\" Location=\"Resource(wss,noXinviewofY_LIST)\" /><ParameterBinding Name=\"NoAnnouncementsHowTo\" Location=\"Resource(wss,noXinviewofY_DEFAULT)\" /></ParameterBindings></View>","Id":"ba84217c-8561-4234-aa95-265081e74be9","ImageUrl":"/_layouts/15/images/generic.png?rev=45","IncludeRootFolder":false,"ViewJoins":null,"JSLink":"clienttemplates.js","ListViewXml":"<View Name=\"{BA84217C-8561-4234-AA95-265081E74BE9}\" DefaultView=\"TRUE\" MobileView=\"TRUE\" MobileDefaultView=\"TRUE\" Type=\"HTML\" DisplayName=\"All Items\" Url=\"/Lists/l2/AllItems.aspx\" Level=\"1\" BaseViewID=\"1\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/generic.png?rev=45\" ><Query><OrderBy><FieldRef Name=\"ID\" /></OrderBy></Query><ViewFields><FieldRef Name=\"LinkTitle\" /></ViewFields><RowLimit Paged=\"TRUE\">30</RowLimit><JSLink>clienttemplates.js</JSLink><XslLink Default=\"TRUE\">main.xsl</XslLink><Toolbar Type=\"Standard\"/></View>","Method":null,"MobileDefaultView":true,"MobileView":true,"ModerationType":null,"NewDocumentTemplates":null,"OrderedView":false,"Paged":true,"PersonalView":false,"ViewProjectedFields":null,"ViewQuery":"<OrderBy><FieldRef Name=\"ID\" /></OrderBy>","ReadOnlyView":false,"RequiresClientIntegration":false,"RowLimit":30,"Scope":0,"ServerRelativePath":{"DecodedUrl":"/Lists/l2/AllItems.aspx"},"ServerRelativeUrl":"/Lists/l2/AllItems.aspx","StyleId":null,"TabularView":true,"Threaded":false,"Title":"All Items","Toolbar":"","ToolbarTemplateName":null,"ViewType":"HTML","ViewData":null,"VisualizationInfo":null}]
...
```
Where the last part is your response.

```
value: [{"Aggregations":null,"AggregationsStatus":null,"BaseViewId":"1","ColumnWidth":null,"ContentTypeId":{"StringValue":"0x"},"CustomFormatter":null,"DefaultView":true,"DefaultViewForContentType":false,"EditorModified":false,"Formats":null,"Hidden":false,"HtmlSchemaXml":"<View Name=\"{BA84217C-8561-4234-AA95-265081E74BE9}\" DefaultView=\"TRUE\" MobileView=\"TRUE\" MobileDefaultView=\"TRUE\" Type=\"HTML\" DisplayName=\"All Items\" Url=\"/Lists/l2/AllItems.aspx\" Level=\"1\" BaseViewID=\"1\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/generic.png?rev=45\"><XslLink Default=\"TRUE\">main.xsl</XslLink><JSLink>clienttemplates.js</JSLink><RowLimit Paged=\"TRUE\">30</RowLimit><Toolbar Type=\"Standard\" /><ViewFields><FieldRef Name=\"LinkTitle\" /></ViewFields><Query><OrderBy><FieldRef Name=\"ID\" /></OrderBy></Query><ParameterBindings><ParameterBinding Name=\"NoAnnouncements\" Location=\"Resource(wss,noXinviewofY_LIST)\" /><ParameterBinding Name=\"NoAnnouncementsHowTo\" Location=\"Resource(wss,noXinviewofY_DEFAULT)\" /></ParameterBindings></View>","Id":"ba84217c-8561-4234-aa95-265081e74be9","ImageUrl":"/_layouts/15/images/generic.png?rev=45","IncludeRootFolder":false,"ViewJoins":null,"JSLink":"clienttemplates.js","ListViewXml":"<View Name=\"{BA84217C-8561-4234-AA95-265081E74BE9}\" DefaultView=\"TRUE\" MobileView=\"TRUE\" MobileDefaultView=\"TRUE\" Type=\"HTML\" DisplayName=\"All Items\" Url=\"/Lists/l2/AllItems.aspx\" Level=\"1\" BaseViewID=\"1\" ContentTypeID=\"0x\" ImageUrl=\"/_layouts/15/images/generic.png?rev=45\" ><Query><OrderBy><FieldRef Name=\"ID\" /></OrderBy></Query><ViewFields><FieldRef Name=\"LinkTitle\" /></ViewFields><RowLimit Paged=\"TRUE\">30</RowLimit><JSLink>clienttemplates.js</JSLink><XslLink Default=\"TRUE\">main.xsl</XslLink><Toolbar Type=\"Standard\"/></View>","Method":null,"MobileDefaultView":true,"MobileView":true,"ModerationType":null,"NewDocumentTemplates":null,"OrderedView":false,"Paged":true,"PersonalView":false,"ViewProjectedFields":null,"ViewQuery":"<OrderBy><FieldRef Name=\"ID\" /></OrderBy>","ReadOnlyView":false,"RequiresClientIntegration":false,"RowLimit":30,"Scope":0,"ServerRelativePath":{"DecodedUrl":"/Lists/l2/AllItems.aspx"},"ServerRelativeUrl":"/Lists/l2/AllItems.aspx","StyleId":null,"TabularView":true,"Threaded":false,"Title":"All Items","Toolbar":"","ToolbarTemplateName":null,"ViewType":"HTML","ViewData":null,"VisualizationInfo":null}]
```